.ibm/pipelines/auth/secrets-rhdh-secrets.yaml

```diff
@@ -32,4 +32,5 @@ data:
   DH_TARGET_URL: aHR0cDovL3Rlc3QtYmFja3N0YWdlLWN1c3RvbWl6YXRpb24tcHJvdmlkZXItc2hvd2Nhc2UtY2kucmhkaC1wci1vcy1hOTgwNTY1MDgzMGIyMmMzYWVlMjQzZTUxZDc5NTY1ZC0wMDAwLnVzLWVhc3QuY29udGFpbmVycy5hcHBkb21haW4uY2xvdWQ=
   GOOGLE_CLIENT_ID: dGVtcA==
   GOOGLE_CLIENT_SECRET: dGVtcA==
+  NAMESPACE: dGVtcA== # Saving the namespace in the secrets to be used as an environment variable within Playwright.
```

Comment:
@gustavolira if the idea is to use `NAMESPACE` in playwright you can use existing `NAME_SPACE` variable that is already exported in `env_variables.sh`, so this will not be needed